server/service/individualServices/AuthorizationApplication.js

```diff
@@ -218,13 +215,15 @@ exports.isOpeartionisExistAsync = async function (applicationName, applicationRe
     }
 }
 
-exports.IsApplicationExists = async function (applicationaName, ReleaseNumber, authorization) {
+exports.IsApplicationExists = async function (applicationaName, releaseNumber,authorization) {
     let isApplicationNameExit = false
-    let isReleaseNumberExit = false
+    let  isReleaseNumberExit = false
+
     try {
         let applicationDataFile = await FileprofileOperation.getApplicationDataFileContent()
         if (applicationDataFile !== undefined) {
             let applicationData = JSON.parse(fs.readFileSync(applicationDataFile, 'utf8'));
+            const IsApplicationExistsIntheConfigFile= await getApplicationandReleaseNumberAtConfigFile(applicationaName,releaseNumber)
```

Comment:
maintain lowerCamelCase in the variable declaration. change IsApplicationExistsIntheConfigFile to isApplicationExistsInTheConfigFile